README.md

```diff
@@ -1,2 +1,283 @@
+}
+
+### 2.3 Get a GraphServiceClient object
+After you have set the correct application ID/client ID, you must get a **GraphServiceClient** object to make requests against the service. The SDK stores the account information for you, but when a user signs in for the first time, it invokes the UI to get the user's account information.
+
+```java
+IGraphServiceClient graphClient =
+  GraphServiceClient
+    .builder()
+    .authenticationProvider(msalAuthenticationProvider)
+    .buildClient();
+```
+
+## 3. Make requests against the service
+
+After you have a GraphServiceClient object, you can begin making calls against the service. The requests against the service look like our [REST API](https://developer.microsoft.com/en-us/graph/docs/concepts/overview).
+
+### 3.1 Get the user's drive
+
+To retrieve the user's drive:
+
+```java
+graphClient
+  .me()
+  .drive()
+  .buildRequest()
+  .get(new ICallback<Drive>() {
+     @Override
+     public void success(final Drive result) {
+        System.out.println("Found Drive " + result.id);
+     }
+     ...
+     // Handle failure case
+  });
+```
+
+### 4 Sample
+
+AndroidManifest.xml
+```
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="your.domain.name">
+
+    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/AppTheme">
+        <activity android:name=".MainActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+
+        <activity android:name="com.microsoft.identity.client.BrowserTabActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW"/>
+                <category android:name="android.intent.category.DEFAULT"/>
+                <category android:name="android.intent.category.BROWSABLE"/>
+                <data android:host="auth" android:scheme="msal<YOUR_CLIENT_ID>"/>
+            </intent-filter>
+        </activity>
+
+    </application>
+</manifest>
+```
+
+MainActivity.java
+```
+package com.your.packagename;
+
+import android.content.Intent;
+import android.os.AsyncTask;
+import android.os.Build;
+import android.os.StrictMode;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
```

Comment:
Again here in code, remove common part from readme, only put the part which is relevant and would be useful for someone using the library.

Reply:
I removed this code, will move it in wiki in next PR.